micro_sam/training/util.py

```diff
@@ -75,8 +75,6 @@ def get_trainable_sam_model(
         checkpoint_path=checkpoint_path,
         return_sam=True,
         return_state=True,
-        use_lora=use_lora,
-        rank=rank,
```

Comment:
I would rather keep these here and remove lines 98-102. Then we don't duplicate the functionality of creating the PEFT SAM wrapper here and in `get_sam_model`.

Reply:
Okay, I went an extended route to take care of a few minor, yet important things I wanted to be careful about. Now it should be GTG. Thanks for the suggestion!